node-manager/src/wallet.rs

```diff
@@ -133,6 +134,14 @@ impl MutinyWallet {
     }
 }
 
+#[allow(unused_must_use)]
```

Comment:
a warning was being throw from just dropping the return of the function so I had to add this